Extensions/Multiplayer/multiplayerobjectruntimebehavior.ts

```diff
@@ -0,0 +1,639 @@
+/*
+  GDevelop - Multiplayer Object Behavior Extension
+  Copyright (c) 2013-2016 Florian Rival (Florian.Rival@gmail.com)
+*/
+
+namespace gdjs {
+  const logger = new gdjs.Logger('Multiplayer');
+  /**
+   * The MultiplayerObjectRuntimeBehavior represents a behavior that can be added to objects
+   * to make them synchronized over the network.
+   */
+  export class MultiplayerObjectRuntimeBehavior extends gdjs.RuntimeBehavior {
+    // Which player is the owner of the object.
+    // If 0, then the object is not owned by any player, so the host is the owner.
+    playerNumber: number = 0;
+
+    // The action to be executed when the player disconnects.
+    actionOnPlayerDisconnect: string;
```

Comment:
We should see if we can use an enum to avoid storing potential long strings on every single instance.